core/tests/webdriverio_utils/ExplorationEditorMainTab.js

```diff
@@ -0,0 +1,903 @@
+// Copyright 2022 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Page object for the exploration editor's main tab, for use in
+ * WebdriverIO tests.
+ */
+
+var forms = require('./forms.js');
+var general = require('./general.js');
+var interactions = require('../../../extensions/interactions/webdriverio.js');
+var ruleTemplates = require(
+  '../../../extensions/interactions/rule_templates.json');
+var waitFor = require('../webdriverio_utils/waitFor.js');
+var action = require('./action.js');
+
+var _NEW_STATE_OPTION = 'A New Card Called...';
+var _CURRENT_STATE_OPTION = '(try again)';
+
+var ExplorationEditorMainTab = function() {
+  /*
+   * Interactive elements
+   */
+  var addResponseDetails = $('.e2e-test-add-response-details');
+  var addResponseHeader = $('.e2e-test-add-response-modal-header');
+  var multipleChoiceAnswerOptions = function(optionNum) {
+    return $(
```

Comment:
No need to use await with the single selectors, we just need to use await with the all selectors.
The test that was failing before was due to the [changes](https://github.com/oppia/oppia/pull/15614#discussion_r906863285) in the wdio.conf.js file not because of not using await with the single selectors. After I reverted the change the test doesn't fail a single time.